p2p/tests/ban.rs

```diff
@@ -15,7 +15,7 @@
 //
 // Author(s): A. Altonen
 #![allow(unused)]
-extern crate test_utils;
+extern crate p2p_test_utils;
```

Comment:
I'm curious why this external crate is needed.

Reply:
I'm not sure. I just renamed `test_utils` crate into `p2p_test_utils`

Reply:
Can you please try to remove it? 🙃